.github/ISSUE_TEMPLATE/breaking_change.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Breaking Change
```

Comment:
Does yaml front-matter in Markdown support yaml comments? If so, probably worth documenting these properties.

Reply:
No idea. I just copied exiting template and edited.
Example at https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms doesn't have comments.
I also don't know how to test this before putting it out there - docs suggest there isn't.